### DISCUSSION TITLE
need help to get github edu

### DISCUSSION THREAD

**Discussion by real-Hafez** [community/community#143899] [GitHub Education]
### Hi everyone,

i  tried so many times and i send my colledge schedule + i send my id + i send transcript but all in arabic cause im from egypt and every time it makes a wrong i try to capture it from phone but still wrong

**Comment by huddmax** [comment]
Hi there! 👋
Contact GitHub Support: If you’re still having issues, reach out to [GitHub Education Support](https://support.github.com/contact/education) and explain the situation. They may be able to review your application manually, given your unique case.
Good luck, and hopefully this helps get your application approved soon! 😊
